src/components/HFUI/HFUI.container.js

```diff
@@ -58,6 +59,9 @@ const mapDispatchToProps = dispatch => ({
   getFeatureFlags: (authToken) => {
     dispatch(WSActions.send(['feature_flags.get', authToken]))
   },
+  getActiveAlgoOrders: () => {
+    dispatch(AOActions.getActiveAlgoOrders({ initialFetch: true }))
```

Comment:
should be initialFetch on each mode.. not just the first time user logs in

Reply:
you're correct, check this one: https://github.com/bitfinexcom/bfx-hf-ui-core/pull/422/commits/8a2fd195d955c238857ea747ffd7c5a745df1c38